docker/seahorn-build.Dockerfile

```diff
@@ -0,0 +1,85 @@
+#
+# Dockerfile for SeaHorn
```

Comment:
the comment on this dockerfile seems wrong.
It does not compile seahorn. 

Perhaps this is the base for seahorn-build image. In which case it is not clear why it is cloning seahorn.
A build image must include all dependencies except for seahorn itself.

Reply:
I thought it would be faster to run `git fetch` instead of clone in the actual build file, which is why I cloned it here.

Reply:
if this is `seahorn-build`, then this is a base build image for all builds of seahorn. At this point, you don't know which version of the code will be build: master, dev, custom branch, etc.

On travis-ci, travis makes the appropriate version of the code available and it has to be imported into the container (you actually have that in TRAVIS specific docker already)